stable/enterprise/templates/_common.tpl

```diff
@@ -247,6 +247,50 @@ app.kubernetes.io/component: {{ $component | lower }}
 {{- end -}}
 
 
+{{/*
+Create an image specification template that can override the default image
+based on global settings
+*/}}
+{{- define "enterprise.common.image" -}}
+{{- if eq (printf "%T" .Values.image) "string" }}
+  {{- .Values.image | trim -}}
+{{- else if and .Values.image.digest -}}
```

Comment:
Is the "and" necessary here? I may be wrong but I don't think the "and" is helpful here.

Reply:
nope, removed.